TestPriorityQueue::test_queue_push_pop_priorities() fails on 32-bit

I was surprised to notice this in Debian test logs on all 32-bit architectures for 2.13.3:

```
=================================== FAILURES ===================================
_______________ TestPriorityQueue.test_queue_push_pop_priorities _______________

self = <tests.test_pqueues.TestPriorityQueue object at 0xf28747b0>

    def test_queue_push_pop_priorities(self):
        temp_dir = tempfile.mkdtemp()
        queue = ScrapyPriorityQueue.from_crawler(
            self.crawler, FifoMemoryQueue, temp_dir, [-1, -2, -3]
        )
        assert queue.pop() is None
        assert len(queue) == 0
        req1 = Request("https://example.org/1", priority=1)
        req2 = Request("https://example.org/2", priority=2)
        req3 = Request("https://example.org/3", priority=3)
        queue.push(req1)
        queue.push(req2)
        queue.push(req3)
        assert len(queue) == 3
        dequeued = queue.pop()
        assert len(queue) == 2
        assert dequeued.url == req3.url
        assert dequeued.priority == req3.priority
>       assert queue.close() == [-1, -2]
E       AssertionError
```

--- a/tests/test_pqueues.py
+++ b/tests/test_pqueues.py
@@ -92,7 +92,7 @@ class TestPriorityQueue:
         assert len(queue) == 2
         assert dequeued.url == req3.url
         assert dequeued.priority == req3.priority
-        assert queue.close() == [-1, -2]
+        assert set(queue.close()) == {-1, -2}
 
 
 class TestDownloaderAwarePriorityQueue:

